File: src/1450_number-of-students-doing-homework-at-a-given-time.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        num_students = 0

        for start_time, end_time in zip(startTime, endTime):
            num_students += start_time <= queryTime <= end_time

        return num_students

File: Elixir/1450_number-of-students-doing-homework-at-a-given-time.ex
defmodule Solution do
  @spec busy_student(start_time :: [integer], end_time :: [integer], query_time :: integer) ::
          integer
  def busy_student(start_time, end_time, query_time) do
    Enum.zip(start_time, end_time)
    |> Enum.count(fn {start_time, end_time} ->
      query_time >= start_time and query_time <= end_time
    end)
  end
end
